### PR TITLE
Modify MySQL dependency scope

### PR DESCRIPTION
**Pull request by CalvinKirs**
Because the license is not in compliance, if you need to use MySQL, you can add it yourself

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -180,6 +180,13 @@
                 <version>${flink.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <!--Because the license is not in compliance, if you need to use MySQL, you can add it yourself-->
```
Comment (zhongjiajie) [inline pom.xml:183]:
```suggestion
            <!--Because the license is not in ASF compliance, if you need to use MySQL, you can add it yourself-->
```

Comment (zhongjiajie) [reply pom.xml:183]: Maybe we should add keyword `ASF` or `Apache-2.0` and something else here?

**Review by zhongjiajie — APPROVED** [review]
Add NIT for mysql dependent comment, and should we also change `doc` repo?

File: pom.xml (hunk 1 above)
Comment (CalvinKirs) [reply pom.xml:183]: Our license is apache 2.0, maybe there is no need to declare again here

**Comment by CalvinKirs** [comment]
> Add NIT for mysql dependent comment, and should we also change `doc` repo?

yup

File: pom.xml (hunk 1 above)
Comment (zhongjiajie) [reply pom.xml:183]: You're right 👍

**Comment by zhongjiajie** [comment]
I going to merge this patch now.

**Pull request merged**